src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -356,8 +356,8 @@
     <comment>The string to be displayed when there is no current value in the y-coordinate number box.</comment>
   </data>
   <data name="Globals_DefaultLaunchPositionCheckbox.Content" xml:space="preserve">
-    <value>Use system default</value>
-    <comment>A checkbox for the "launch position" setting. Toggling this control sets the launch position to the system default.</comment>
+    <value>Let system position window</value>

```

Comment:
Since system == Windows, isn't this like "Let Windows position window"? 😄
I wonder if something like "Let Windows decide" would be better. We already use such a label for "DefaultWindowsConsoleName" after all...

Reply:
I'm 100% down for "Let Windows decide"

Reply:
fwiw I chose "let system position window" because:

<img width="412" alt="image" src="https://user-images.githubusercontent.com/189190/208200168-34b04178-649a-4f1c-bf25-b98d312051bd.png">

This is okay as well :)